blocks/code/code.css

```diff
@@ -0,0 +1,350 @@
+/* NOTE: this is a modified version of the css from prism.js
```

Comment:
I've marked the relevant sections with the coment `/* MODIFIED: ... */` so you can find them easily with cmd-f. I chose not to raise a second PR since it makes the test url not refelct the final result. Also, this way anyone modifying the file already knows what's been modified without having to look back and forth in PRs. The tradeoff is we need to be disciplined about maintaining the comments.